pinot-common/src/main/java/org/apache/pinot/sql/parsers/SqlNodeAndOptions.java

```diff
@@ -23,17 +23,25 @@
 
 
 public class SqlNodeAndOptions {
+  private String _sql;
   private final SqlNode _sqlNode;
   private final PinotSqlType _sqlType;
   // TODO: support option literals other than STRING
-  private final Map<String, String> _options;
-
+  private final Map<String, String> _queryOptions;
+  private final Map<String, String> _debugOptions;
```

Comment:
Let's not add debug options here. We don't want to separate debug options and query options because it is quite hard to use.

Reply:
it is here for backward compatibility reasons. let me add a todo to remove it after 0.11 release